package.json

```diff
@@ -1,152 +1,146 @@
-{
```

Comment:
are we having linting issues? should we bring back prettier? 😞

Reply:
Hahahaha yes probably